app/views/questions/index.html.erb

```diff
@@ -148,5 +166,36 @@
 
     </div>
   </div>
+
+    <!-- Defines how the modal will look -->
+  <div class="modal fade" id="social_display_Modal" role="dialog" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <!-- Sets the style of the modal header  -->
+        <div class="modal-header" style="padding:35px 50px; background-color: #00b3fe; color: #FFFFFF">
+          <!-- Close button -->
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <!-- Displays the question at the top of the modal. Javascript finds the "modal-title" class and populates it with the question contents. -->
+          <h4 class="modal-title"></h4>
+          </div>
+          <div class="modal-body" style="padding:40px 50px;">
```

Comment:
Ditto for style -- let's move it into a class defined in a stylesheet and reference the class here.